group_vars/build.yml

```
@@ -1,4 +1,29 @@
 ---
+# Admin Tools
+admin_packages:
+  - nano
+  - vim
+  - htop
+  - strace
+  - jq
+  - iftop
+  - wget
+  - curl
+  - tmux
+  - git
+  - nmap
+  - tcpdump
+  - net-tools
+  - unzip
+  - tmpwatch
+  - rclone
+  - "{{ byobu if ansible_distribution_major_version < '9' | default(omit) }}"
```

```suggestion
  - "{{ 'byobu' if ansible_distribution_major_version < '9' else omit }}"
```

Same story as [here](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/998#discussion_r1397117191).

---

Not resolved :(